apps/sms/test/marionette/share_activity_test.js

```diff
@@ -51,7 +51,6 @@ marionette('Messages as share target', function() {
 
         // Exit from activity and verify that Messages is dismissed
         messagesApp.performHeaderAction();
-        messagesApp.selectAppMenuOption('Delete Draft');
```

Comment:
so, we save a draft first before opening the view, so when we open the view we load the draft; then when we exit the activity we don't have the dialog because we haven't edited the draft.

I figured it was not necessary to fix this change because of your patch to remove the dialogs anyway.